gamegenerator.py

```diff
@@ -300,7 +302,7 @@ def __postConversionForRetropie__(self):
         dosboxCfg = open(os.path.join(self.getLocalGameOutputDir(), "dosbox.cfg"), 'a')
         # add mount c at end of dosbox.cfg
         romsFolder = util.getRomsFolderPrefix(self.conversionType, self.conversionConf)
-        retropieGameDir = romsFolder + "/" + self.genre + "/" + self.game + ".pc" if self.useGenreSubFolders else romsFolder + "/" + self.game + ".pc"
+        retropieGameDir = romsFolder + "/" + self.genre + "/" + self.gameDir if self.useGenreSubFolders else romsFolder + "/" + self.gameDir
```

Comment:
this should never be used for retropie so this is not needed

Reply:
> this should never be used for retropie so this is not needed

only for consistency